src/app/infokit/infokit.component.ts

```diff
@@ -7,25 +7,26 @@ import { InfokitService } from '../services/infokit.service';
     styleUrls: ['./infokit.component.scss']
 })
 export class InfokitComponent implements OnInit {
+    private static _localStorageKey = 'pcprepkitUser';
     // sets infokit visible/Invisible Default : Invisible
     public infokitstate = false;
+    public infokitAvailable = false;
     /**
      * sets the Infokit Information to visible/Invisible Default : Invisible.
      * When Infokit Information is Invisible the Infokit Selector is Visible and Vice Versa.
      */
     public showInfo = false;
     // sets the visiblility of the Icons in Infokit Selector
-    public infokitactive = {
-        malaria_def: {value: false, def: 'Malaria'},
-        pc_policy: {value: false, def: 'Peace Corps Policy'},
-        animation: {value: false, def: 'Animation'},
-        do_dont: {value: false, def: 'Do\'s and Don\'ts'},
-        odd_one_out: {value: false, def: 'Odd One Out'},
-        match_meds: {value: false, def: 'Side Effects'},
-        side_effects: {value: false, def: 'Match the Meds'},
-        doctor_info: {value: false, def: 'Doctor Information'},
-    };
-
+    public infokitactive = [
```

Comment:
Angular can't loop a dictionary by using ngFor, in the previous code I used a pipe with switch statements to convert the dictionary into an array with all those values, Since you suggested a better way by removing the switch statements, the pipe was converting a dictionary into an array. I removed the pipe as it was now unnecessary for this and had async issues.